it/connectors/DepartureConnectorSpec.scala

```diff
@@ -201,7 +200,7 @@ class DepartureConnectorSpec
         LocalDateTime.now
       )
 
-      val response = HateoasResponseDeparture(departure)
```

Comment:
This seems to be in the v1/phase 4 code - did you mean to touch this? The file and class doesn't seem to be renamed (and we should keep it that way really), the only P4 stuff that should be touched are the routes file changes.

Reply:
Well caught - I did a refactoring which affected many more files than I thought it would and forgot to change these back, thanks!